src/clib-build.c

```diff
@@ -627,14 +602,37 @@ int main(int argc, char **argv) {
     memcpy((void *)opts.dir, dir, size);
   }
 
+  if (!root_package) {
+    const char *name = NULL;
+    char *json = NULL;
+    unsigned int i = 0;
+
+    do {
+      name = manifest_names[i];
+      json = fs_read(name);
```

Comment:
Doesn't look like we're cleaning up this memory. We should `free(json)` in the `if` below.

Reply:
fixed!